doc/docusaurus/docusaurus-examples.cabal

```diff
@@ -24,6 +24,28 @@ common ghc-version-support
   if (impl(ghc <9.6) || impl(ghc >=9.7))
     buildable: False
 
+library docusaurus-code
+  import:         lang, ghc-version-support
+  hs-source-dirs: static/code
+
+  if (impl(ghcjs) || os(windows))
+    buildable: False
+
+  other-modules:
+    AuctionValidator
+    BasicPlutusTx
+    BasicPolicies
+    BasicValidators
+
+  build-depends:
+    , base               >=4.9   && <5
+    , plutus-core        ^>=1.31
+    , plutus-ledger-api  ^>=1.31
+    , plutus-tx          ^>=1.31
```

Comment:
Does the release script know that these need to be updated?

Reply:
Good point - I don't think so. I'll remove these bounds

Reply:
No I was wrong, it actually does know.